src/notation/inotationinteraction.h

```diff
@@ -240,7 +240,7 @@ class INotationInteraction
     virtual void addLyricsVerse() = 0;
 
     // Text navigation
-    virtual void navigateToLyrics(MoveDirection direction) = 0;
+    virtual void navigateToLyrics(MoveDirection direction, bool moveOnly = false) = 0;
```

Comment:
Here you have `bool moveOnly = false`, but in `notationinteraction.h` you have `bool moveOnly = true`. That's scary...

There are people who totally ban the use of default argument values in virtual methods for this reason... I don't think we should go that far, but we must be careful.